src/main/java/uk/ac/kcl/rowmappers/DocumentRowMapper.java

```diff
@@ -53,17 +53,19 @@
     ApplicationContext context;
 
     // mandatory properties required to perform record mapping
-    @Value("${source.srcTableName}")
-    private String srcTableName;
-    @Value("${source.srcColumnFieldName}")
-    private String srcColumnFieldName;
-    @Value("${source.primaryKeyFieldName}")
-    private String primaryKeyFieldName;
     @Value("${source.primaryKeyFieldValue}")
     private String primaryKeyFieldValue;
     @Value("${source.timeStamp}")
     private String timeStamp;
 
+    // optional fields, required in docman profile
```

Comment:
At some point we may want to generalise this to include paths for file objects in an objectstore

Reply:
Agree. We should also revisit the `docman` profile -- for now, I would keep this profile for backwards compatibility, but as deprecated. We should revisit how the documents are being read and processed from the filesystem and/or general objectstore.